src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade.component.html

```diff
@@ -21,30 +27,44 @@
              [ngStyle]="healthData.health.status | healthColor"
              [ngbPopover]="healthChecks"
              popoverClass="info-card-popover-cluster-status">
-             {{ healthData.health.status | healthLabel | uppercase }}
+            {{ healthData.health.status | healthLabel | uppercase }}
           <i *ngIf="healthData.health?.status !== 'HEALTH_OK'"
              class="fa fa-exclamation-triangle"></i>
         </div>
       </div>
-      <div class="col-lg-3 h-50 d-flex flex-column border justify-content-center align-items-center">
-        <span class="bold">Upgrade Status</span>
-        <ng-container *ngIf="upgradeInfo.versions.length > 0; else noUpgradesAvailable">
-          <span class="mt-2"
-                i18n>
-          <i [ngClass]="[icons.up]"
-             class="text-info"></i>
-            Upgrade available</span>
-          <div i18n-ngbTooltip
-               [ngbTooltip]="(healthData.mgr_map | mgrSummary).total <= 1 ? 'To upgrade, you need minimum 2 mgr daemons.' : ''">
-            <button class="btn btn-accent mt-2"
-                    id="upgrade"
-                    aria-label="Upgrade now"
-                    [disabled]="(healthData.mgr_map | mgrSummary).total <= 1"
-                    i18n>Upgrade now</button>
-          </div>
+    </cd-card>
+
+    <cd-card class="col-sm-3 px-3 d-flex"
+             aria-label="Upgrade Status"
+             id="upgradeStatus"
+             *ngIf="{upgradeInfo: upgradeInfo$ | async, error: upgradeInfoError$ } as upgrade">
```

Comment:
If I put it in a parent class it would prevent the other cards from loading since this is only needed in two area's. But this won't call twice since I am [sharing the pipe returns](https://www.learnrxjs.io/learn-rxjs/operators/multicasting/sharereplay) in other areas so it'll only gets called once and the results gets shared across the wherever its called.